### PR TITLE
Amls 4515

### PR DESCRIPTION
**Pull request by samzaroughmrc**
porting the frontend to latest protoype version - where the user status is set in text rather relying on colors and image

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: test/controllers/RegistrationProgressControllerSpec.scala [hunk 1]
```diff
@@ -333,8 +332,8 @@ class RegistrationProgressControllerSpec extends AmlsSpec
             status(responseF) must be(OK)
 
             val submitButtons = Jsoup.parse(contentAsString(responseF)).select("button[type=\"submit\"]")
-            submitButtons.size() must be(1)
-            submitButtons.first().hasAttr("disabled") must be(true)
+            submitButtons.size() must be(0)
+//            submitButtons.first().hasAttr("disabled") must be(true)
```
Comment (ajchilts) [inline test/controllers/RegistrationProgressControllerSpec.scala]: Can this be deleted?

File: test/views/registrationprogress/registration_progressSpec.scala [hunk 2]
```diff
@@ -38,35 +39,15 @@ class registration_progressSpec extends AmlsSpec with MockitoSugar with AddressG
...
-            addressGen.sample.get,
-            Seq.empty,
-            true
-          )
-
-        doc.select("#section1-status").text mustBe Messages("progress.visuallyhidden.view.amend")
-        doc.select("#section2-status").text mustBe Messages("progress.visuallyhidden.view.started")
-
-      }
+      // the first h2.heading-small should be info about progress section
```
Comment (matthewharris1) [inline test/views/registrationprogress/registration_progressSpec.scala]: Can this be removed?

File: test/views/renewal/renewal_progressSpec.scala [hunk 3]
```diff
@@ -48,51 +48,69 @@ class renewal_progressSpec extends AmlsSpec with MustMatchers{
...
 
-    "show intro for non MSB and TCSP businesses" in new ViewFixture {
+    "show ready to submit renewal when information are completed" in new ViewFixture {
 
-      override def view = views.html.renewal.renewal_progress(Seq.empty, false, false, readyForRenewal)
+      override def view = views.html.renewal.renewal_progress(Seq.empty, false, true, readyForRenewal, true)
 
-      html must include (Messages("renewal.progress.tponly.intro", DateHelper.formatDate(renewalDate)).convertLineBreaks)
+      doc.select("#renewal-information-completed").get(0).text() must be(Messages("renewal.progress.information.completed.info"))
+//      html must include (Messages("renewal.progress.ready.to.submit.intro"))
```
Comment (matthewharris1) [inline test/views/renewal/renewal_progressSpec.scala]: Can this be uncommented?

File: test/controllers/RegistrationProgressControllerSpec.scala (hunk 1 above)
Comment (samzaroughmrc) [reply test/controllers/RegistrationProgressControllerSpec.scala]: done - thanks

File: test/views/registrationprogress/registration_progressSpec.scala (hunk 2 above)
Comment (samzaroughmrc) [reply test/views/registrationprogress/registration_progressSpec.scala]: done - thanks

File: test/views/renewal/renewal_progressSpec.scala (hunk 3 above)
Comment (samzaroughmrc) [reply test/views/renewal/renewal_progressSpec.scala]: done - thanks

File: app/views/include/registration_progress_sidebar.scala.html [hunk 4]
```diff
@@ -16,21 +16,33 @@
...
+        <div class="business-info__item">
+            <h2 class="heading-small">Your services</h2>
+            @if(serviceNames.length == 1) {
+                <p>@Html(serviceNames(0))</p>
+            } else {
+                <ul class="list list-bullet">
+                    @serviceNames.map(name => Html("<li>" + name + "</li>"))
+                </ul>
+            }
+            <p style="margin-top: -0.75em;">
```
[the remark this answers is not in the document]
Comment (samzaroughmrc) [reply app/views/include/registration_progress_sidebar.scala.html]: done, thanks

**Pull request merged**